custom_components/scout_alarm/binary_sensor.py

```diff
@@ -94,17 +109,19 @@ def device_state_attributes(self):
         return {
             ATTR_ATTRIBUTION: ATTRIBUTION,
             "device_id": self._device['id'],
-            "battery_low": self._device['reported']['battery'].get('low'),
             "device_type": self._device['type'],
+            "battery_low": self._device['reported']['battery'].get('low')
         }
 
     @property
     def device_info(self):
         """Return device registry information for this entity."""
         return {
             "identifiers": {(DOMAIN, self._device['id'])},
-            "manufacturer": "Scout Alarm",
-            "name": self.name
+            "manufacturer": self._device['reported']['manufacturer'],
+            "name": self.name,
+            "sw_version": self._device['reported']['fw_version'],
+            "model": self._device['reported']['model']
```

Comment:
My devices don't have a `model` attribute. Since python hashes throw when a key doesn't exist via `[]`, can we change this to `.get('model')` instead?